code/modules/admin/topic.dm

```diff
@@ -426,7 +426,6 @@
 		to_chat(GLOB.banlist_savefile["minutes"], minutes)
 		to_chat(GLOB.banlist_savefile["bannedby"], usr.ckey)
 		GLOB.banlist_savefile.cd = "/base"
-		feedback_inc("ban_edit",1)
```

Comment:
I see all the ban-related counters got axed? Why? Not saying it's a bad thing, just feels odd.

Reply:
We can query the ban DB for more accurate statistics